lib/datasource.js

```diff
@@ -597,7 +597,7 @@ DataSource.prototype.define = function defineClass(className, properties,
   var args = slice.call(arguments);
 
   if (!className) {
-    throw new Error('Class name required');
+    throw new Error(g.f('Class name required'));
```

Comment:
@superkhau `Class name` is not a variable like `className `; I don't think if we should skip it? thought?